docs/modules/WebDriver.md

```diff
@@ -24,13 +24,13 @@ To run Selenium Server you will need Java and Chrome or Firefox browser installe
 
 ### PhantomJS
 
-PhantomJS is a headless alternative to Selenium Server that implements
+PhantomJS is a [headless browser](https://en.wikipedia.org/wiki/Headless_browser) alternative to Selenium Server that implements
```

Comment:
> Is it really a good idea to have hundreds of lines of documentation inside the source code?

Yes. At least you always have one source of thruth :+1: 

> I'm losing track of my commits ;-) Could you merge them and/or invoke the markdown-conversion manually?

I will lose track too. Could you just copy your changes to docblocks of WebDriver and Recorder classes, and I will merge it